README.md

```diff
@@ -1 +1,101 @@
 # SpatialConnect
+
+#### Fresh install of EFC and the associated dependencies. ####
+
+#### **Dependencies List** ####
+What you will need to have installed prior to running the program:
+
+##### Git repositories #####
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
+* [React-native-maps](https://github.com/boundlessgeo/react-native-maps)
+* [React-native-spatialconnect](https://github.com/boundlessgeo/react-native-spatialconnect)
+
+##### Signing apps #####
+
+First you will need to obtain the signing key file and passwords from one of the project
+leaders.  Email spatialconnect@boundlessgeo.com to request it.
+
+
+##### General #####
+
+
+* Homebrew:
+``
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+``
+
+* Node:
+``
+brew install node
+``
+
+* Watchman:
+``
+brew install watchman
+``
+
+* React-native CLI:
+``
+npm install -g react-native-cli
+``
+
+* Flow:
+``
+npm i flow-bin@ --save-dev
+``
+
+
+###### iOS specific ######
+* Xcode
+* Carthage
+* Fabric
+* Crashlytics.framework
+
+###### Android specific ######
+* Android Studio
+* Java SE Runtime 1.7
```

Comment:
You can remove this line.  A JDK includes a JRE so it makes sense to say "install the jdk" but not install both a JDK and a JRE